### DISCUSSION TITLE
DFP-34X-2C2 EEPROM dump question

### DISCUSSION THREAD

**Discussion by Voidstranger** [Anime4000/RTL960x#177] [Q&A]
Does someone have DFP-34X-2C2 EEPROM dump?

Ethtool -e command doesn't work for me.

**Comment by Anime4000** [comment]
I think someone have done that somewhere in issue post

**Comment by tantien11** [comment]
This is a dump from working stick, i tried to flash it to another stick but it doesn't work, don't know why 😬

[dfp-34x-2c2.bin](https://drive.google.com/file/d/1SQUsYhgnjddlE2DuHG35BxvVL02iIoNL/view?usp=drivesdk)

**Reply by tantien11** [reply]
Or may be you can try flash firmware via uart 
download .tar and extract it, 
`setenv sw_version0 V1.0-220414`
`setenv sw_active 0`
`setenv sw_commit 0`
`saveenv`
`printenv` to find address of kernel and rootfs (`fl_kernel1=`, `fl_rootfs1=`)
`xf <address of kernel>` and tranfer uImage via xmodem
press `n` and press `enter` when it starting count down
`xf <address of rootfs>` and tranfer rootfs via xmodem
press `n` and hope it will work  

ps: i'm using Xshell to access UART and tranfer files via xmodem

**Comment by Gradius2** [comment]
After trying to downgrade the FW by using web gui, my XPON just went blinked.

Tried UART and getting nothing too. Will order a SOP to DIP adapter and try to use an EEPROM programmer. However, if tantien11 says already tried to flash with that .bin and didn't work, so I guess I just "burned" my XPON for good.
